honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,173 @@
+"""ModifierSet Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import ModifierBase, Void
+
+
+class BaseModifierSet(IDdBaseModel):
+    """Base class for the sets assigned to Faces."""
+
+    # TODO: What to use for default? Void or None?
```

Comment:
The default should definitely be `None`. Setting it to `Void` might have the same result from a Radiance perspective but it will force our Python libraries to treat it like a real modifier rather than as a non-existent one.